packages/documentation-site/patternfly-docs/content/accessibility/helper-text/helper-text.md

```diff
@@ -82,5 +84,20 @@ The following HTML attributes and PatternFly classes can be used for more fine-t
 | `id` | `.pf-v6-c-helper-text__item` | Sets the `id` attribute of the helper text item, which should be passed to the main element's `aria-describedby` attribute if the `.pf-v6-c-helper-text` element is not given an `id` instead. |
 | `aria-hidden="true"` | `.pf-v6-c-helper-text__item-icon > i` | Removes the helper text item icon from the accessibility tree, preventing assistive technologies from potentially announcing duplicate or unnecessary information without visually hiding it. **Required**. |
-| `.pf-v5-screen-reader` | `.pf-v6-c-helper-text__item.pf-m-dynamic > span` | Renders visually hidden text that is accessible only to assistive technologies such as a screen reader. **Required** if the helper text item is intended or expected to contain static text and always be visible, but have dynamic styling. Typically this should convey the status type of the helper text item. |
- 
\ No newline at end of file
+| `.pf-v6-screen-reader` | `.pf-v6-c-helper-text__item.pf-m-dynamic > span` | Renders visually hidden text that is accessible only to assistive technologies such as a screen reader. This should be included if the helper text item is intended or expected to contain static text and always be visible, but have dynamic styling, or if the helper text item content doesn't convey a status. Typically this should convey the status type of the helper text item. |
+ 
+## Additional considerations
+
+Consumers must ensure they take any additional considerations when customizing an accordion, using it in a way not described or recommended by PatternFly, or in various other specific use-cases not outlined elsewhere on this page.
+
+### Visually hidden text
+
+While we recommend including visually hidden text for helper text items, it's important to understand more about the when and why. Typically we recommend including visually hidden text within the helper text item for 2 scenarios:
+
+- the helper text item content is static, but the styling and status will dynamically change
+- the helper text item content does not clearly convey any type of status
+
+For the first scenario, if helper text item content will always remain static, any visual styling changes themselves won't be conveyed to users. Including visually hidden text within the helper text item can help convey that change in status and styling when navigating to the helper text item, but also via `aria-live` announcements. Without this visually hidden text, the same text content would be announced without any change at best, or nothing would be announced at worst in the case of `aria-live`.
+
+For the second scenario, it can be difficult for some users to understand the difference between messages of different status types depending on the wording. Including visually hidden text that helps convey that status can help differentiate between the two, or better understand whether a single helper text item is crucial to continuing through a workflow or not.
+
```

Comment:
Makes sense, though I'd also bring up localization as a consideration. It is something we use a lot in Cockpit so if there was a mix of two languages it would be strange